maintainers/maintainer-list.nix

```diff
@@ -6703,6 +6703,12 @@
     name = "Sebastian Neubauer";
     keys = [ { fingerprint = "2F93 661D AC17 EA98 A104  F780 ECC7 55EE 583C 1672"; } ];
   };
+  Flameopathic = {
```

Comment:
This should be a separate commit.

Reply:
handled in separate PR, addition no longer necessary; removed it from this PR :+1: